.github/workflows/publish.yml

```diff
@@ -43,23 +43,30 @@ jobs:
       uses: ./.github/actions/anchor
       with:
         run: |
-          # deploy cargo
+          sed -i 's/^version = ".*"$/version = "$(basename ${{ github.ref }})"/' rust-sdk/${{ matrix.package }}/Cargo.toml
+          # TODO: replace workspace dependencies with fixed version
+          yarn build rust-sdk/${{ matrix.package }} --output-style static
+          cd rust-sdk/${{ matrix.package }} && cargo publish --allow-dirty
+      env:
+        CRATES_IO_API_TOKEN: ${{ secrets.CRATES_TOKEN }}
 
   verifiable:
-    strategy:
-      matrix:
-        program: []
     runs-on: ubuntu-latest
     steps:
     - name: Checkout Repository
       uses: actions/checkout@v4
       with:
         fetch-depth: 0
-    - name: Deploy cargo
-      uses: ./.github/actions/anchor
+    - name: Verifiable build
+      run: anchor build --verifiable
```

Comment:
Is it needed to use `--verifiable`?
In my understanding, Anchor verifiable build is not updated and we are using new one from Ellipsis-Labs.
https://github.com/Ellipsis-Labs/solana-verifiable-build
(Of course I believe that tech behind them is same (use some prebuild docker env to build), but I think the image is different)

If the purpose of this build is getting IDL, IDL is not affected from build environment.

Reply:
This step doesn't do anything that we need to rely on. All it does now is add the .so binary and idl to the github release